### ISSUE TITLE
Update object-path to 0.11.6

### ISSUE THREAD

**Issue by gesabehrends** [tokuda109/next-useragent#373]
Could you please update the object-path version by merging this PR https://github.com/tokuda109/next-useragent/pull/372. 
Next-useragent has a moderate security issue because of this.

https://github.com/advisories/GHSA-v39p-96qg-c8rf

**Comment by sinchang** [comment]
It seems the below packages are not being used in code. Should we remove them? cc @tokuda109 

```
"kind-of": ">=6.0.3",
"minimist": ">=1.2.5",
"node-fetch": ">=2.6.1",
"object-path": "0.11.5",
```

**Comment by tokuda109** [comment]
@gesabehrends @sinchang 
Fixed in version 2.7.0
